src/utilities/MessageFormatter.ts

```diff
@@ -140,6 +141,11 @@ export const convertExtension = (file: Asset | undefined): string => {
   }
 };
 
+export const getAbsoluteFilePath = (path: string) => {
+  if (Platform.OS === 'ios') return path;
+  return 'file://' + path;
```

Comment:
I don't think we should check it by `Platform`, it could be wrong
Do something like:
```
return path?.startsWith?.('file:/') ? path : `file://${path}`
```
And remove the `Platform` check above

Reply:
I updated the function. Please check